-- Add FreeBSD support to device_query crate (x11 input, build config, module selection).
--- cargo-crates/device_query-1.1.3/Cargo.toml.orig	2026-09-27 23:28:22 UTC
+++ cargo-crates/device_query-1.1.3/Cargo.toml
@@ -31,7 +31,7 @@ version = "0.3.9"
 [build-dependencies.pkg-config]
 version = "0.3.9"
 
-[target."cfg(target_os = \"linux\")".dependencies.x11]
+[target."cfg(any(target_os = \"linux\", target_os = \"freebsd\"))".dependencies.x11]
 version = "2.17.2"
 features = ["xlib"]
 
